<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1536</width>
    <height>1181</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="DirectoryA">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>0</y>
      <width>1171</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="BrowseB">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>169</width>
      <height>43</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="bstRB">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>930</y>
      <width>271</width>
      <height>38</height>
     </rect>
    </property>
    <property name="text">
     <string>Binary Search Tree</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="tstRB">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>980</y>
      <width>301</width>
      <height>38</height>
     </rect>
    </property>
    <property name="text">
     <string>Ternary Search Tree</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="tRB">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>1030</y>
      <width>197</width>
      <height>38</height>
     </rect>
    </property>
    <property name="text">
     <string>Trie</string>
    </property>
   </widget>
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>100</y>
      <width>400</width>
      <height>800</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="StopWordB">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>169</width>
      <height>43</height>
     </rect>
    </property>
    <property name="text">
     <string>StopWord</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="StopWordA">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>50</y>
      <width>1171</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="cmd">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>891</width>
      <height>801</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="cmdInput">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>980</y>
      <width>891</width>
      <height>43</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>930</y>
      <width>771</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Type your command in here:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>1020</y>
      <width>169</width>
      <height>43</height>
     </rect>
    </property>
    <property name="text">
     <string>Build Tree</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="avlCheck">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>940</y>
      <width>160</width>
      <height>38</height>
     </rect>
    </property>
    <property name="text">
     <string>Make Avl</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="hashRb">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>1030</y>
      <width>197</width>
      <height>38</height>
     </rect>
    </property>
    <property name="text">
     <string>Hash</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1536</width>
     <height>46</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuInvertedIndex">
    <property name="title">
     <string>InvertedIndex</string>
    </property>
   </widget>
   <addaction name="menuInvertedIndex"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
